pkg/local/sorted_queue.go

```diff
@@ -0,0 +1,28 @@
+package local
+
+// A SortedQueue implements heap.Interface and holds strings.
+type SortedQueue []string
```

Comment:
So why do all the methods use `any` and not `string`?

Reply:
Required by the heap interface